test/new_relic/slack_notifications_tests/slack_notifier_tests.rb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative '../../../.github/workflows/scripts/slack_notifications/slack_notifier'
+
+class SlackNotifications < Minitest::Test
+  def test_send_slack_message_zero_args
+    assert_raises(ArgumentError) { SlackNotifier.send_slack_message() }
+  end
+
+  def test_send_slack_message_too_many_args
+    assert_raises(ArgumentError) {
+      SlackNotifier.send_slack_message("I am a notification message!", "But I'm one too many")
+    }
+  end
+
+  def test_send_slack_message
+    SlackNotifier.stub(:sleep, nil) do
+      HTTParty.stub(:post, nil) do
+        assert_nil SlackNotifier.send_slack_message("I am a notification message!")
+      end
+    end
+  end
+
+  def test_exit_handler_no_fail
+    SlackNotifier.stub(:sleep, nil) do
+      HTTParty.stub(:post, nil) do
+        SlackNotifier.send_slack_message("I am a notification message!")
+        assert_empty SlackNotifier.errors_array
+      end
+    end
+  end
+
+  def test_exit_handler_fail
```

Comment:
I don't see the case with more than one error being tested in this file. Would you mind adding that case?

Reply:
With that method split in two, multiple errors are now being tested with `[test_errors_array_multiple_errors](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-30d914ac7129feb2d13bf43e052cf29fe91564381ac3623355ac7f02ae77d096R49-R56)` and `[test_report_errors_multiple_errors](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-30d914ac7129feb2d13bf43e052cf29fe91564381ac3623355ac7f02ae77d096R71-R79)`